.github/CODEOWNERS

```diff
@@ -95,6 +95,7 @@
 /sigsci/                                 @signalsciences info@signalsciences.com
 /shoreline/                              @ytnytn1 support@shoreline.io
 /sleuth/                                 @mtbnut pleal@sleuth.io
+/superwise/                              @oryan-omer support@superwise.ai
```

Comment:
Small nit but this should go after `/storm/` not after `/sleuth/`.

Reply:
Fixed @yzhan289